src/IntegrationHeader/IntegrationHeader.jsx

```diff
@@ -18,7 +18,7 @@ export const IntegrationHeader = ({
     alignItems='center'
     mb={[6, 8]}>
     <Typography variant='titleLarge'>{title}</Typography>
-    <Typography variant='textSmall' color='outline' display={['none', 'block']}>
+    <Typography variant='textSmall' color='outline' display='block'>
```

Comment:
It seems we need some responsive layout decisions here, I don't think on Mobile Portrait view the `Reachtag PHONE - Schützt Ihr Telefon vor Verlust` as title and this link fits in one line.

Please use the **storybook** to recreate this scenario.

Reply:
@elenawq is this fixed?

Reply:
I need @alsofronie to review it too